### PR TITLE
Contributor names now appear where they should

### PR DESCRIPTION
**Pull request by asalani93**
Addresses #2067.

### CODE REVIEW

File: app/controllers/data_sets_controller.rb [hunk 1]
```diff
@@ -72,7 +72,10 @@ def create
       return
     end
 
-    if @cur_user.nil?
+    if !session[:contributor_name].nil?
```
Comment (bdonald25) [inline app/controllers/data_sets_controller.rb]: if session[:contributor_name]?

Comment (asalani93) [inline app/controllers/data_sets_controller.rb]: apparently that is not valid ruby

Comment (bdonald25) [inline app/controllers/data_sets_controller.rb]: unless session[:contributor_name].nil?

Comment (asalani93) [inline app/controllers/data_sets_controller.rb]: That makes rubocop throw up if you have an elsif or else following it.

**Pull request merged**